docs/userguide/modules/creating-service-bindings/pages/custom-path-injection.adoc

```diff
@@ -132,3 +132,91 @@ metadata:
+resource (CR) for both the API groups: `binding.operators.coreos.com` and `servicebinding.io`.
+
+The specification also provides a way of defining exactly where binding data needs to be projected
+into, which may work in situations where the above methods won't work.  This is done by defining
+`ClusterWorkloadResourceMapping.servicebindings.io` resources, which define where binding data gets
+projected for a given workload kind.  This works for both CoreOS and specification service binding
+resources.
+
+.Example Schema of the `ClusterWorkloadResourceMapping` resource
+[source,yaml]
+----
+apiVersion: servicebinding.io/v1alpha3
+kind: ClusterWorkloadResourceMapping
+metadata:
+  name: # string <1>
+  ...
+spec:
+  versions: # required, at least one entry must be defined
+  - version: # string <2>
+    annotations: # string (Fixed JSONPath), optional <3>
+    containers: # optional
+    - path: # string (JSONPath) <4>
+      name: # string (Fixed JSONPath), optional <5>
+      env: # string (Fixed JSONPath), optional <6>
+      volumeMounts: # string (Fixed JSONPath), optional <7>
+    volumes: # string (Fixed JSONPath), optional <8>
+----
+<1> Name of this resource, which must be qualified as `plural.group`.  For example, `cronjobs.batch` for CronJobs.
+<2> Version of the resource we're mapping.  Any version not specified can be matched with the "*" wildcard.
+<3> Optional: where annotations live, specified with a fixed JSONPath.  The default value is `.spec.template.spec.annotations`.
+<4> Optional: where containers live, specified with a JSONPath.  If no entries are defined, the {servicebinding-title} defaults to two paths: `.spec.template.spec.containers` and `.spec.template.spec.initContainers` with all other fields set as their default).
+<5> Optional: Identifier of the `.name` field in a pod.  The default value is `.name`.
+<6> Optional: Identifier of `.env` environment variables.  The default value is `.env`.
+<7> Optional: Identifier of the volume mounts of a pod lives.  The default value is `.volumeMounts`.
+<8> Optional: where volumes live, specified with a fixed JSONPath.  The defaults value is `.spec.template.spec.volumes`.
+
+[IMPORTANT]
+====
+In this context, a "Fixed JSONPath" is a subset of the JSONPath grammar that accepts only the
+following operations:
+
+. Field lookup: `.spec.template`
+. Array indexing: `.spec['template']`
+
+All other operations are forbidden.
+====
+
+[NOTE]
+====
+Most of these fields are optional, and SBO assumes PodSpec-compatible defaults when not specified.
+SBO requires that each of these fields is structurally equivalent to the corresponding field in a Pod
+deployment.  For example, the contents of the `.env` field in a workload resource must be able to
+accept the same structure of data that the `.env` field in a Pod resource would.  Otherwise,
+projecting binding information into such a workload may result in unexpected behavior from SBO.
```

Comment:
```suggestion
* Most of these fields are optional, and when not specified, the {servicebinding-title} assumes defaults that are compliant with the `PodSpec workload resources.
* The {servicebinding-title} requires that each of these fields is structurally equivalent to the corresponding field in a pod deployment.  For example, the contents of the `.env` field in a workload resource must be able to accept the same structure of data that the `.env` field in a Pod resource would.  Otherwise, projecting binding data into such a workload might result in unexpected behavior from the {servicebinding-title}.
```

Reply:
Fixed.